### ISSUE TITLE
Loader docs return value for source from transform?

### ISSUE THREAD

**Issue by jonaskello** [nodejs/loaders#35]
I think the most current docs for the loader hooks are here:

https://github.com/nodejs/node/blob/master/doc/api/esm.md#loadurl-context-defaultload

In the section above in the table for formats, it has a column called 

> Acceptable types For source returned by resolve or load

This leads me to believe that `resolve` somehow would be capable of returning `source` but I think it is not? Is this an error in the docs or have I misunderstood something?

**Comment by GeoffreyBooth** [comment]
It should probably say acceptable values for `format`; @JakobJingleheimer ?

**Comment by jonaskello** [comment]
The format is specified in the first column so I think it the resolve part should just be removed?

Acceptable types For source returned by ~resolve or~ load

**Comment by JakobJingleheimer** [comment]
>It should probably say acceptable values for `format`; @JakobJingleheimer ?

@GeoffreyBooth no, the 3rd column is indeed types for `source`. That table is saying when `format` is foo, `source` can be bar. I'm not sure how much value the table provides (it existed before though) as in most circumstances, the choice is made for you.

>Acceptable types For source returned by ~resolve or ~load

Yes! `resolve` does not return a `source` property. Thank you for the catch @jonaskello. PR incoming.

**Comment by JakobJingleheimer** [comment]
nodejs/node#40234
